helper/testhelpers/teststorage/teststorage.go

```diff
@@ -102,7 +98,10 @@ func MakeRaftBackendWithConf(t testing.T, coreIdx int, logger hclog.Logger, extr
 		"performance_multiplier": "8",
 	}
 	for k, v := range extraConf {
-		conf[k] = v
+		val, ok := v.(string)
```

Comment:
What motivated this change?

Reply:
Param changing from `map[string]string` into `map[string]interface{}`. This was part of adding `PhysicalFactoryConfig` for backends. We are not using it anymore since we stopped supporting autopilot HCL configs, but kept the changes in as its might turn out to be useful in future.